README.md

```diff
@@ -33,6 +33,23 @@ In case of error try
 service netfilter-persistent save  
 service netfilter-persistent reload
 ```
+Now, install dependencies using `glide`.
+```
+export PATH=$PATH:$GOPATH/bin
+curl https://glide.sh/get | sh
+glide install
+glide update
+```
+Build glutton,
+```
+cd $GOPATH/src/github.com/mushorg/glutton
+go install app/server.go
```

Comment:
`go build` also does the same as `go install`, but from what I've read, `go build` doesn't cache anything. `go install` caches non-main packages so that the next compile happens faster. `go run` is for running the executable of the main package.